doc/tasks.rst

```diff
@@ -500,6 +500,27 @@ private/hidden tasks
 
 If task name starts with an underscore '_', it will not be included in the output.
 
+.. literalinclude:: tutorial/private_task.py
+
+.. code_block:: console
+
+    $ doit
+    .  hello
+    $ doit dumpdb
```

Comment:
I do not think this is a good example. The output is very lengthy and the `dumpdb`  is an advanced command that most users would not know it exist or its purpose.

Better would be to have a private task that prints out something into the console. So you could see that it was executed although its name does not show up. 

what do you think?